core/internal.xml

```diff
@@ -160,6 +160,306 @@
+		</setters>
+		<rules>
+			<stat name="strength" value="-1" alt="Ability Score Decrease" />
+		</rules>
+	</element>
+
+	<element name="Dexterity" type="Ability Score Improvement" source="Internal" id="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI_DEXTERITY_DECREASE_1">
+		<supports>Custom Ability Score Decrease 1</supports>
+		<description>
+			<p>Your Dexterity decreases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="allow duplicate">true</set>
+		</setters>
+		<rules>
+			<stat name="dexterity" value="-1" alt="Ability Score Decrease" />
+		</rules>
+	</element>
+
+	<element name="Constitution" type="Ability Score Improvement" source="Internal" id="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI_CONSTITUTION_DECREASE_1">
+		<supports>Custom Ability Score Decrease 1</supports>
+		<description>
+			<p>Your Constitution decreases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="allow duplicate">true</set>
+		</setters>
+		<rules>
+			<stat name="constitution" value="-1" alt="Ability Score Decrease" />
+		</rules>
+	</element>
+
+	<element name="Intelligence" type="Ability Score Improvement" source="Internal" id="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI_INTELLIGENCE_DECREASE_1">
+		<supports>Custom Ability Score Decrease 1</supports>
+		<description>
+			<p>Your Intelligence decreases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="allow duplicate">true</set>
+		</setters>
+		<rules>
+			<stat name="intelligence" value="-1" alt="Ability Score Decrease" />
+		</rules>
+	</element>
+
+	<element name="Wisdom" type="Ability Score Improvement" source="Internal" id="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI_WISDOM_DECREASE_1">
+		<supports>Custom Ability Score Decrease 1</supports>
+		<description>
+			<p>Your Wisdom decreases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="allow duplicate">true</set>
+		</setters>
+		<rules>
+			<stat name="wisdom" value="-1" alt="Ability Score Decrease" />
+		</rules>
+	</element>
+
+	<element name="Charisma" type="Ability Score Improvement" source="Internal" id="ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI_CHARISMA_DECREASE_1">
+		<supports>Custom Ability Score Decrease 1</supports>
+		<description>
+			<p>Your Charisma decreases by 1.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="allow duplicate">true</set>
+		</setters>
+		<rules>
+			<stat name="charisma" value="-1" alt="Ability Score Decrease" />
+		</rules>
+	</element>
+
+	<!-- Sizes -->
+	<element name="Tiny" type="Racial Trait" source="Internal" id="ID_INTERNAL_RACIAL_TRAIT_TINY">
+		<description>
+			<p>Your size is Tiny.</p>
+		</description>
+		<sheet display="false" />
+		<compendium display="false" />
+		<rules>
+			<grant type="Size" id="ID_SIZE_TINY" />
+		</rules>
+	</element>
+
+	<element name="Small" type="Racial Trait" source="Internal" id="ID_INTERNAL_RACIAL_TRAIT_SMALL">
+		<description>
```

Comment:
Maybe add a support for "Size Standard" or "Size SM" that just has small and medium

Reply:
I don't think we need to, using IDs is preferable when there's not a giant list of options.